src/components/quote/quote.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { ReactNode } from 'react';
+import styled from 'styled-components';
+import { TextStyles } from '../typography/typography-v2';
+import { theme } from '../layout/theme';
+
+interface QuoteProps {
+  children: ReactNode | ReactNode[];
+}
+
+const QuoteText = styled.p`
```

Comment:
`blockquote` is the correct semantic tag to use for quotes. See here https://developer.mozilla.org/en-US/docs/Web/HTML/Element/blockquote.